mne/chpi.py

```diff
@@ -402,7 +403,11 @@ def _setup_hpi_struct(info, model_n_window,
     else:  # == 'multipole'
         coils = _prep_mf_coils(info)
     scale = make_ad_hoc_cov(info, verbose=False)
-    scale = _get_whitener_data(info, scale, meg_picks, verbose=False)[0]
+
+    scalings = _handle_default('scalings', None)
```

Comment:
you shouldn't need to do this -- this is only used for cov rank internally by `compute_whitener`, which will easily pass for this diagonal matrix even if it's not a perfect match for the values `make_ad_hoc_cov` uses.